api/src/services/opportunities/search_opportunities.py

```diff
@@ -22,8 +21,10 @@ def search_opportunities(
 
     sort_fn = asc if search_params.sorting.is_ascending else desc
 
-    stmt = select(Opportunity).order_by(
-        sort_fn(getattr(Opportunity, search_params.sorting.order_by))
+    stmt = (
+        select(Opportunity)
+        .order_by(sort_fn(getattr(Opportunity, search_params.sorting.order_by)))
+        .where(Opportunity.is_draft.is_(False))  # Only ever return non-drafts
```

Comment:
This needs to be added to the `api/src/services/opportunities/get_opportunities.py` select query as well in order to keep `is_draft == False` opportunities from being shared in the GET endpoint.

Reply:
I'm assuming that at some point we'll be building out the ability to modify drafts, and we'd need the GET endpoint to return them, but we'll use whatever auth we build out to restrict whether someone can see a draft opportunity or not?

Reply:
My current understanding is that the drafts should not be viewable to the general public, and that they would only be viewable by the grant org admins. Providing any drafts would require authenticating to the current system, which I imagine at this point is pretty far in the future.